### DISCUSSION TITLE
[Looking for feedback] Feed Sticky Announcements

### DISCUSSION THREAD

**Discussion by evi-liu** [community/community#35830] [Other Feature Feedback, Questions, & Ideas]
👋 Hi folks, I'm Evi and I'm the Product Manager for the GitHub Feed. 

Today, we posted a sticky announcement on the “For you” feed. It is a pilot to a larger effort we’re undertaking to surface all announcements from projects/repositories you follow into a central, digestible space on your feed, like this:

<img width="1000" alt="image" src="https://user-images.githubusercontent.com/8809849/195168544-8f5f6226-6801-4be5-81e6-019ac81555e9.png">

Of course, this feed is _for you_. Like other features on the feed, you can filter out announcements by using “Filter your feed” found on the upper right corner. 

We're aiming to have this available in a couple of weeks. But for now, any feedback or thoughts on this direction?

**Comment by Amolith** [comment]
I would like to hide this column entirely. I very intentionally don't follow anyone or star/watch repositories because I personally find that all the extra activity clutters the UI with unnecessary noise. This new column exacerbates the issue by enabling a firehose of content by default.

**Comment by arpadgabor** [comment]
Hi, the section is okay, but I keep getting shown announcements that I dismissed already.

![image](https://user-images.githubusercontent.com/9270494/203022126-032b68ef-cb24-4a8d-a711-8034b1752676.png)

The announcement section keeps showing me announcements that I already dismissed multiple times. Sometimes it also shows duplicate announcements while I'm in the process of dismissing them.

PS: A dismiss all would be useful.
